bootstrap-salt.sh

```diff
@@ -5162,17 +5162,21 @@ __configure_freebsd_pkg_details() {
     fi
     FROM_FREEBSD="-r FreeBSD"
 
-    ## add saltstack freebsd repo
-    salt_conf_file=/usr/local/etc/pkg/repos/saltstack.conf
-    {
-        echo "SaltStack:{"
-        echo "    url: \"${SALTPKGCONFURL}\","
-        echo "    mirror_type: \"http\","
-        echo "    enabled: true"
-        echo "    priority: 10"
-        echo "}"
-    } > $salt_conf_file
-    FROM_SALTSTACK="-r SaltStack"
+    ##### Workaround : Waiting for SaltStack Repository to be available for FreeBSD 12 ####
+    if [ 12 -ne $( freebsd-version -k | cut -d. -f1 ) ]; then
```

Comment:
```suggestion
    if [ ${DISTRO_MAJOR_VERSION} -ne 12 ]; then
```

@sticky-note You can re-use already calculated FreeBSD release variable.

Reply:
<3